.github/workflows/Cleanup Caches by a branch.yml

```diff
@@ -9,6 +9,10 @@ on:
     branches: [main, 'ODS-*']
     types: [ closed ]
 
+concurrency:
```

Comment:
This workflow executes only when a PR is closed, so there will not be an in progress workflow to cancel. This change can be reverted.

Reply:
Revert it back . Thanks